modules/basis_universal/SCsub

```diff
@@ -22,7 +22,7 @@ tool_sources = [
 	"basisu_resample_filters.cpp",
 	"basisu_resampler.cpp",
 	"basisu_ssim.cpp",
-	"basisu_tool.cpp",
+	# "basisu_tool.cpp", # Excluded, unused and it contains the main which can cause compile error due to duplicated symbol.
```

Comment:
Could you `git rm thirdparty/basis_universal/basisu_tool.cpp` and document its exclusion in the readme?